includes/struct_def.h

```diff
@@ -5,7 +5,7 @@
 # define PROMPT "> "
 # define SPACE ' '
 # define PIPE '|'
-# define ENV_VAR '$'
+# define DOLLAR_SIGN '$'

```

Comment:
ok you kinda missed my point but nevermind 😂

Reply:
Also, I'll change dollar sign to something else once we come up with something we both like ! I just did it and then saw your comment =)